src/chain_access.rs

```diff
@@ -1,11 +1,11 @@
-use crate::errors::ChainSyncError;
+use crate::errors::RuntimeError;
 use bitcoin::{Script, Transaction, Txid};
 use esplora_client::blocking::BlockingClient;
 use lightning::chain::{Confirm, Filter, WatchedOutput};
 use std::sync::{Arc, Mutex};
 
 #[allow(dead_code)]
-pub(crate) struct LipaChainAccess {
```

Comment:
We should revert `pub(crate)` here and below, shouldn't we?

Reply:
Absolutely, thanks! https://github.com/getlipa/lipa-lightning-lib/pull/52/commits/e874b97993063e56f6dcc17df7f658bb0fc7bb28